index.js

```diff
@@ -22,6 +24,19 @@ app.use(morgan('combined'));
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
+app.use(session({
+  secret: 'team vidar',
```

Comment:
Please consider keeping the secret and key in a .env file

Reply:
Alright. Thanks